tests/perp_test.py

```diff
@@ -1,47 +1,47 @@
 # perp_test.py
+from _pytest.python_api import approx
 from grpc._channel import _InactiveRpcError
+from pytest import raises
 
-import tests
-from nibiru import common
+from nibiru import Coin, common
+from tests import dict_keys_must_match, transaction_must_succeed
 
 PRECISION = 6
 
 
-class TestPerp(tests.ModuleTest):
-    def test_open_close_position(self):
+class TestPerp:
```

Comment:
Do we still need a class if we don't use unittest anymore?

Reply:
Removed.